src/io.cpp

```diff
@@ -358,35 +302,37 @@ PngPipelineInput::PngPipelineInput(cairo_surface_t *cairoSurface, Camera camera,
     image.height = cairo_image_surface_get_height(cairoSurface);
 }
 
-PipelineInputList PromptPngPipelineInput() {
+PipelineInputList GetPngPipelineInput(const PipelineOptions &values) {
     // I'm not sure why, but i can't get an initializer list to work here. Probably something to do
     // with copying unique ptrs
     PipelineInputList result;
     cairo_surface_t *cairoSurface = NULL;
-    std::string pngPath;
-
+    std::string pngPath = values.png;
+    
     while (cairoSurface == NULL || cairo_surface_status(cairoSurface) != CAIRO_STATUS_SUCCESS) {
```

Comment:
This while loop was here originally because every iteration of the while loop, it would prompt for a new name. But with the new CLI, we should just attempt to create the surface, then `exit(1)` if `cairo_surface_status(cairoSurface) != CAIRO_STATUS_SUCCESS`